src/instructlab/configuration.py

```diff
@@ -311,6 +318,17 @@ def api_base(self):
         """Returns server API URL, based on the configured host and port"""
         return get_api_base(self.host_port)
 
+    @model_validator(mode="before")
+    def fill_defaults(
+        cls, values: typing.Dict[str, typing.Any]
+    ) -> typing.Dict[str, typing.Any]:
+        defaults = {
+            "model_path": DEFAULTS.DEFAULT_MODEL,
+            "llama_cpp": {"gpu_layers": -1, "max_ctx_size": 4096, "llm_family": ""},
```

Comment:
This seems to repeat what is defined in the attribute defaults, e.g. for `llama_cpp`, the default used is already listed in line 302. The other listed fields have defaults too. What does this function (and other similar functions here) give that the `= <default_value> / Field(lambda: <default_value>)` wouldn't be able to achieve?